tests/test_async_processing_export.py

```diff
@@ -256,6 +256,44 @@
     }
 }
 
+process_chain_short_stac = {
+    1: {
+        "module": "g.region",
+        "inputs": {
+            "raster": "elevation@PERMANENT"
+        },
+        "flags": "p",
+        "verbose": True
+    },
+    2: {
```

Comment:
Just a hint that this is the old PC style and we want to replace it in the future. See https://github.com/mundialis/actinia_core/issues/245 (no need to adjust it in this PR)